lib/u3d/unity_versions.rb

```diff
@@ -135,7 +135,7 @@ def save_package_size(version, url)
             size = Integer(response['Content-Length'])
           end
           if size
-            INIparser.create_linux_ini(version, size)
+            INIparser.create_linux_ini(version, size, url)
```

Comment:
The thing is that we are creating and saving an INI object, so I think it makes more sense stating clearly that we create one, and load it later through the UnityVersionDefinition. One thing that would bother me if we would go that way is that UnityVersionDefinition is more than just the ini.